styles/lastfm/catppuccin.user.css

```diff
@@ -1701,18 +1709,24 @@
       .mask-image(url("/static/images/icons/date_picker_icon.1d7b21ee3a36.svg"), @accent-color, 0 0, 14px 24px, no-repeat);
     }
 
-    .masthead-logo {
-      .mask-image(url("/static/images/logo_static.adb61955725c.png"), @red, 50% 50%, skip, no-repeat);
+    // .masthead-logo {
+    //   .mask-image(url("/static/images/logo_static.adb61955725c.png"), @red, 50% 50%, skip, no-repeat);
+    // }
+    // .masthead-logo-loading {
+    //   .mask-image(url("/static/images/logo_animate@2x.e93e5b7db3de.gif"), @red, 50% 50%, contain, no-repeat);
+    // }
```

Comment:
Do we need to keep this as a comment?

Reply:
> Do we need to keep this as a comment?

I'm not sure if the redesign is permanent or an A/B experiment. I guess it can be removed, we can simply rollback to a previous version if the redesign is scrapped.